MAINTAINING.md

```diff
@@ -155,6 +155,25 @@ GitHub's review sidebar.
 If there are multiple maintainers reviewing a PR, always give the other
 maintainers reasonable time to ACK before dismissing their review.
 
+### Release, Feature Freeze, and Backports
+
+Before the official release of a new RIOT version, two feature freeze periods
+are announced on the
+[RIOT development email list](https://lists.riot-os.org/mailman/listinfo/devel):
+The soft feature freeze and the hard feature freeze. During the soft feature
+freeze only PRs with minor impact should be merged into master. The hard feature
+freeze begins when the release manager creates a new release branch. Therefore,
+the restriction on merging PRs into the master branch are lifted at that point.
+
+Once the release branch is created, no new features should be backported.
+Instead, backports should consist only of bug fixes or of reverting features
+that were added during the last development period (and not part of any
+release), but didn't reach the required maturity or API stability yet. For
+bigger changes (which explicitly includes any revert), the PR has to be
+announced in the maintainer mailing list and should be merged no sooner than
+48h after the announcement and needs at least to ACKs. In case of security
+relevant fixes, the PR can be merged 1h after the announcement.
```

Comment:
Do you mean just fixes or also reverts?

Reply:
In the security relevant part? Than yes, I think tying the hands of someone waiting to hit the merge button on a critical security fix is a bad idea.

In the announcement part? I'd say only "high impact" bug fixes should be announced.

Reply:
> In the security relevant part? Than yes, I think tying the hands of someone waiting to hit the merge button on a critical security fix is a bad idea.

I meant the security relevant part. Should security relevant reverts (e.g. a feature broken beyond repair) also be under that rule. If yes, please adopt the sentence so it is clearer.

> In the announcement part? I'd say only "high impact" bug fixes should be announced.

I think security bugs don't need to be announced to maintainers, unless they are so big it should involve some more pairs of eyes than just two (but I guess that is what you are saying).

Reply:
I think I adapted the wording to be clearer. Please hit "unresolve" if you disagree.